app/views/madmin/fields/belongs_to/_show.html.erb

```diff
@@ -1,3 +1,8 @@
 <% if (object = field.value(record)) %>
-  <%= link_to Madmin.resource_for(object).display_name(object), Madmin.resource_for(object).show_path(object), class: "text-blue-500 underline" %>
+  <% begin %>
+    <% associated_resource = Madmin.resource_for(object)%>
+    <%= link_to associated_resource.display_name(object), associated_resource.show_path(object) %>
+  <% rescue Madmin::MissingResource => error %>
```

Comment:
I wonder how this could be handled without using errors for flow control 🤔 It is very possible this is one of the nicest approaches available to us given the structure of the library. I don't mean that in a bad way, just that every code structure will have certain problems that are not easy to solve.

The nice thing about this approach is that it solves the problem exactly where it is occurring from the UI point of view. Rather than having to have a bunch of conditions and rendering branches internally, a missing resource anywhere in the render code is rendered in a helpful way.